package.json

```diff
@@ -158,7 +158,7 @@
         "jest": "^29.7.0",
         "jest-bench": "^29.4.1",
         "jest-circus": "~27.2.3",
-        "jest-cli": "~29.7.0",
+        "jest-cli": "~28.1.3",
```

Comment:
any reason to downgrade this?

Reply:
it said it wasnt found when I was running yarn. Dependabot updated it so I reverted to be safe

Reply:
ill see if I can un-downgrade it